packages/codeeditor/src/lineCol.tsx

```diff
@@ -94,6 +95,7 @@ class LineFormComponent extends React.Component<
    * Render the LineFormComponent.
    */
   render() {
+    const textInputId = DOMUtils.createDomID() + '-line-number-input';
```

Comment:
Can you please move `textInputId` generation into the constructor to ensure it is not regenerated on each render?

Reply:
Thanks for the review. I have made these changes.